Writerside/topics/introduction.md

```diff
@@ -1,97 +1,82 @@
-
-## Expected Outcomes
-
-By the end of this thesis, we anticipate achieving the following outcomes:
-
-1. A memory-efficient reader and handling library for GTFS schedules.
-2. A performant public transit router package based on the RAPTOR algorithm.
-3. An integrating public transit service with REST API.
-4. A user-friendly web application that empowers users to explore transit schedules, plan routes, and assess travel
-   time-based accessibility.
-
-The components will be made available under a permissive or copyleft open-source license.
-
-## Initial Situation
-
-The existing literature highlights several complexities associated with transit routing:
-
-1. **Network Complexity**: Public transit networks are intricate, with multiple routes, stops, footpaths and
-   interconnections. Finding optimal paths requires considering various factors such as transfer times, dwell times, and
-   service frequencies.
-
-2. **Real-Time Constraints**: Passengers expect real-time information on transit availability and delays. Incorporating
-   live data into routing algorithms is essential for accurate predictions.
-
-3. **Scalability**: As cities grow, transit networks expand, leading to scalability challenges. Traditional algorithms
-   struggle to handle large-scale networks efficiently.
-
-## Round-Based Approach
-
-Our research builds upon the work presented in the "Round-Based Public Transit Routing" paper by
-Microsoft. This paper introduces a novel approach that organizes transit routes into rounds, simplifying the routing
-process. Each round represents a set of interconnected trips, allowing for efficient traversal through the network.
-
-## Delimitation
-
-We are not creating a commercial product and are independent of any company interests. This work is conducted as part
-of the Master of Advanced Studies program in Software Engineering at the Eastern Switzerland University of Applied
-Sciences (OST).
+The primary research question addressed by this thesis is how the RAPTOR algorithm can be implemented and validated
+within the context of the Swiss public transit network. This question is particularly relevant given the complexity and
+scale of the network, which poses significant challenges for routing algorithms. A related research question focuses on
+the optimization of the RAPTOR algorithm in Java and C++ and comparing the performance of both programming environments
+for such a task.
+
+These research questions are framed within the context of the current state of public transit networks. The complexity
+of these networks, which involve multiple routes, stops, footpaths, and transfer points, necessitates the development of
+routing algorithms that account for factors such as transfer times, service frequencies, and real-time constraints. The
+research seeks to address these challenges while also considering scalability, as expanding transit systems further
```

Comment:
I don't get this sentence... expanding in what manner?

Reply:
I was thinking of multi-modal or more trip related filters, but I removed it for now.

0a3dc1b